community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
++++
+
+toc=true
+
++++
+
+# [Name Option 1] Workplace Harassment - Code of Conduct 
+# [Name Option 2] An informal guide to avoid and deal with events of harassment and bullying
+# [Name Option 3] Workplace Harassment - An informal guide
+
+## Some basic fundamentals
+
+1. Humans are flawed and make mistakes on a regular basis while discovering new ways to hurt new people.
+
+2. We don't live in an ideal world, so we are not dealing with ideal scenarios.
+
+3. But the above statement doesn't provide wind to the practicality argument. Practicality argument is an exceptionally dangerous proposition in itself. So any time any particular statement is backed up by "being practical", it should take care of few things :-
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
```

Comment:
Highly agree for harassment related things. If one person is feeling harassed, it doesn't matter if others think it's okay.